Civi/Api4/Action/WorkflowMessageExample/Get.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+
+namespace Civi\Api4\Action\WorkflowMessageExample;
+
+use Civi\Api4\Generic\BasicGetAction;
+use Civi\Api4\Generic\Result;
+use Civi\WorkflowMessage\Examples;
+
+class Get extends BasicGetAction {
+
+  /**
+   * @var \Civi\WorkflowMessage\Examples
+   */
+  private $_scanner;
+
+  public function _run(Result $result) {
+    if ($this->select !== [] && !in_array('name', $this->select)) {
+      $this->select[] = 'name';
+    }
+    parent::_run($result);
+  }
+
+  protected function getRecords() {
+    $this->_scanner = new Examples();
+    $all = $this->_scanner->findAll();
+    foreach ($all as &$example) {
+      $example['tags'] = !empty($example['tags']) ? \CRM_Utils_Array::implodePadded($example['tags']) : '';
```

Comment:
Why would we want to return a serialized string from the API? Everywhere else in APIv4 we are careful to *un*serialize strings and always return arrays.

Reply:
Right, this is the same issue as the [other comment about  `data_type`+`serialize`](https://github.com/civicrm/civicrm-core/pull/21338#discussion_r701097710) - these things need to match. It might get confusing if the discussion of this gets split over two subthreads, so I'll close this one.